Resources/Prototypes/Entities/Constructible/Walls/fireaxe_cabinet.yml

```diff
@@ -0,0 +1,48 @@
+﻿- type: entity
+  id: FireAxeCabinet
+  name: fire axe cabinet
+  description: There is a small label that reads \"For Emergency use only\" along with details for safe use of the axe. As if.
+  components:
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Sprite
+      sprite: Constructible/Misc/fireaxe_cabinet.rsi
+      state: cabinet-filled-closed
+    - type: ItemCabinet
+      doorSound: /Audio/Machines/machine_switch.ogg
```

Comment:
netsync false?

Reply:
Sure i'll update extinguisher cabinet as well with this.